cmd/neofs-cli/internal/common/exit.go

```diff
@@ -28,15 +28,11 @@ func ExitOnErr(cmd *cobra.Command, errFmt string, err error) {
 		aclDenied
 	)
 
-	var (
-		code int
-
-		internalErr = new(sdkstatus.ServerInternal)
-		accessErr   = new(sdkstatus.ObjectAccessDenied)
-	)
+	var code int
```

Comment:
single `var` -> multi `var` does not relate to this commit

Reply:
no, ofc, no. but do we need to have 6 line change if it may be 1-line removal (related to any style changes)?

Reply:
gofmt will change other line and var group will become obsolete. I don't mind to not remove braces, but imo code became more laconic

Reply:
ok, up to you. just wanted to highlight that i spent some time looking at changes and see what exactly was removed and what just moved

Reply:
sry bout that mate